contents/blog/welcome-to-deskhog.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "DeskHog is here"
+date: 2025-07-21
+author:
+  - danilo-campos
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/deskhog_2888c77766.jpg
+featuredImageType: full
+featuredVideo: https://youtu.be/lqeD2vvAA4w
```

Comment:
@joethreepwood @daniloc we need to use the no cookie version of youtube URL here

Reply:
```suggestion
featuredVideo: https://www.youtube-nocookie.com/embed/lqeD2vvAA4w
```